sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -26,6 +26,41 @@ Use the client library for App Configuration to create and manage application co
 </dependency>
 ```
 
+### Default HTTP Client
+By default all client libraries ship without any baked-in HTTP client. Users who want to use a client library are encouraged to opt-in to the relevant HTTP client for their needs. For most situations, this is achieved by including the following dependency in the Maven pom.xml file:
+
```

Comment:
I agree. I would put it after the "This will automatically" sentence below, with something like "This will automatically configure all client libraries on the same classpath to make use of Netty for the HTTP client. Netty is the recommended HTTP client for most applications. OkHttp is recommended only when the application being built is deployed to Android devices."

Reply:
Updated

Reply:
updated